### ISSUE TITLE
{:target="_blank"} in front of every link provided in the contributing.md

### ISSUE THREAD

**Issue by zahyaah** [crweiner/hacktoberfest-swag-list#625]
#### When clicked on the any of the links, it should open up in a new tab. 
<img width="1053" alt="image" src="https://user-images.githubusercontent.com/72253920/194042065-5b39cd45-1941-4ff8-a3b8-7c369f020dd3.png">

![image](https://user-images.githubusercontent.com/72253920/194042285-7aea648f-98b0-4f03-bc61-c2b35e192e77.png)

![image](https://user-images.githubusercontent.com/72253920/194042520-a674be04-ae82-4fbd-beb8-8988af0022ab.png)

**Comment by zahyaah** [comment]
I tried the following, but I'm not able to get the required output. 
<img width="927" alt="image" src="https://user-images.githubusercontent.com/72253920/194049532-7656f8b9-9964-4210-ba52-9e7f574fd086.png">

**Comment by zahyaah** [comment]
So, shall I remove the `{:target="_blank"}` from the contributing.md file?

**Comment by Mou-Ikkai** [comment]
> So, shall I remove the `{:target="_blank"}` from the contributing.md file? @Mou-Ikkai @crweiner

No! Because that'd remove the ability for it to open a new tab in the website! x3

**Comment by zahyaah** [comment]
@crweiner Thank you so much for explaining me : )

**Comment by crweiner** [comment]
Hello again @zahyaah and @Mou-Ikkai! 

I just added a new Python extension to the repo that'll automatically add the `target=_"blank"` attribute when MkDocs/Netlify builds the live site without having to specify it manually anymore. 🎉 See PR #659 .

So, now the .md files in this repo are much easier to read, as GitHub will not leave in the `{:target="_blank"}` after the URLs when browsing on GitHub.com. 

Keep in mind that GitHub still won't actually open any links in new tabs, but now you wont see an ugly `{:target="_blank"}` after each link if you're browsing the repo from here.

So much cleaner now here on GitHub and your IDE of choice! 😄 Let me know if you have any questions!
